app/src/main/java/com/nutomic/syncthingandroid/model/Folder.java

```diff
@@ -37,7 +37,7 @@
     public boolean disableSparseFiles;
     public boolean disableTempIndexes;
     public boolean paused;
-    public boolean useLargeBlocks;
+    public boolean useLargeBlocks = false;
     public int weakHashThresholdPct = 25;
```

Comment:
Why default false? I think go now has defaults set to true?

Reply:
I recall this is still unreleased and planned for 1.1.0, so I set it to false until we need it.

Reply:
AFAIK the most recent released beta of 0.10.17 contains 1.0.1, so this change will make it into 0.10.18, which will be released with 1.1.0 - so defaulting to true as well should be good.

Reply:
True